mantidproject/mantid
# NSIS version 3 compatibility

**Description of work**

We have typically had to use NSIS <= v2 to build our Windows installers. v2 has not been maintained for a long time and we have been falling behind.

This work makes the necessary modifications to the NSIS template so that it can work with all versions.

**To test:**

Run the windows build with `BUILD_PACKAGE` and confirm that you get a package out that installs. This confirms we still work with v2.

Download [NSIS v3](http://nsis.sourceforge.net/Download) and install it. Make sure `C:\Program Files (x86)\NSIS` is in the `PATH`. Run-rerun cmake and set `ENABLE_CPACK=ON`. In Visual studio build the package target and verify you get an `.exe` out in the build directory.

*No issue number*

**Release Notes** 
*Internal change. Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
